test/specs/rules/require-trailing-comma.js

```diff
@@ -35,6 +35,22 @@ describe('rules/require-trailing-comma', function() {
         output: 'var t = {\n\ta: 1,\n\tb: 2, };'
     });
 
+    reportAndFix({
+        name: 'missing comma in object pattern',
+        rules: rules,
+        errors: 1,
+        input: 'const { foo, bar } = baz;',
+        output: 'const { foo, bar, } = baz;'
+    });
+
+    reportAndFix({
+        name: 'missing comma in array pattern',
+        rules: rules,
+        errors: 1,
+        input: 'const [ foo, bar ] = baz;',
```

Comment:
I could just be misunderstanding the diff/changes here, but it seems like `input` and `output` are swapped in this.